6_3_iterable.js

```diff
@@ -0,0 +1,25 @@
+class Group {
+    constructor(arr) {
+        this.cluster = arr;
+    }
+    add = value => (this.cluster.includes(value) ? null : this.cluster.push(value));
+    delete = value =>
+        this.cluster.includes(value) ? this.cluster.splice(this.cluster.indexOf(value), 1) : null;
+    has = value => this.cluster.some(item => item === value);
+
+    static from(arr) {
+        return new Group(arr);
```

Comment:
You don't need so kind of simple static methods which only creating a new instance of that class. Consider, that both examples below should have exactly the same results:
```
const data = [1, 2, 3];

const group1 = new Group(data);
const group2 = Group.from(data);
```